client/src/components/EstatePage.js

```diff
@@ -21,7 +26,37 @@ export default function EstatePage() {
   }, []);
 
   function renderProptery() {
-    return properties.map((property) => {
+    return properties.filter((property) => {
+      if(search === "") {
+        return property
+      } else if (property.City.toLowerCase().includes(search.toLowerCase())){
+        return property
+      
+      }}).filter((property) => {
+        if(search1 === "" && search2 === "" ) { return true }
+        else{
+        if (Number(search2) > 0) {
```

Comment:
you should just convert once to Number and store it in a variable, so you don't have to use all the time `Number()`

you can do it at the beginning of the filter function